lib/extensions/ar_order.rb

```diff
@@ -18,8 +22,12 @@ def columns_for_distinct(columns, orders) #:nodoc:
                 end
               end
               # Remove any ASC/DESC modifiers
-              s.gsub(/\s+(?:ASC|DESC)\b/i, "")
```

Comment:
the comment above should be updated to reflect we're removing ASC/DESC modifier unless there's an ORDER we want to retain